services/cinema_hall.py

```diff
@@ -0,0 +1,17 @@
+from django.db.models import QuerySet
+from db.models import CinemaHall
+
+
+def get_cinema_halls() -> QuerySet:
```

Comment:
You should specify iiner type of list
```suggestion
def get_cinema_halls() -> QuerySet[CinemaHall]:
```